XCode 12 on macOS Catalina requires C99 for vsnprintf

Building abcBridge isn't working on macOS Catalina in newer xcode versions, because of XCode's [stdio.h](https://opensource.apple.com/source/Libc/Libc-1353.41.1/include/stdio.h.auto.html) guarding `vsnprintf` and friends behind `_C99_SOURCE`:

```c
#if __DARWIN_C_LEVEL >= 200112L || defined(_C99_SOURCE) || defined(__cplusplus)
__BEGIN_DECLS
int	 snprintf(char * __restrict __str, size_t __size, const char * __restrict __format, ...) __printflike(3, 4);
int	 vfscanf(FILE * __restrict __stream, const char * __restrict __format, va_list) __scanflike(2, 0);
int	 vscanf(const char * __restrict __format, va_list) __scanflike(1, 0);
int	 vsnprintf(char * __restrict __str, size_t __size, const char * __restrict __format, va_list) __printflike(3, 0);
int	 vsscanf(const char * __restrict __str, const char * __restrict __format, va_list) __scanflike(2, 0);
__END_DECLS
#endif /* __DARWIN_C_LEVEL >= 200112L || defined(_C99_SOURCE) || defined(__cplusplus) */
```

However, `_ANSI_SOURCE` is currently being set on macOS: https://github.com/GaloisInc/abcBridge/blob/cb943e93de0d38bcf59faec0c60739e222fb4adb/abcBridge.cabal#L76-L92

This causes:

```
In file included from cbits/abcBridgeGiaDup.c:1:0: error: 

In file included from abc-build/src/aig/aig/aig.h:34:0: error: 

In file included from abc-build/src/misc/vec/vec.h:33:0: error: 

abc-build/src/misc/vec/vecStr.h:700:14: error:
     error: implicitly declaring library function 'vsnprintf' with type 'int (char *, unsigned long, const char *, struct __va_list_tag *)' [-Werror,-Wimplicit-function-declaration]
        nAdded = vsnprintf( Vec_StrLimit(p), nSize, format, args );
                 ^
    |
700 |     nAdded = vsnprintf( Vec_StrLimit(p), nSize, format, args );
    |              ^

abc-build/src/misc/vec/vecStr.h:700:14: error:
     note: include the header <stdio.h> or explicitly provide a declaration for 'vsnprintf'
    |
700 |     nAdded = vsnprintf( Vec_StrLimit(p), nSize, format, args );
    |              ^
2 warnings and 1 error generated.
`gcc' failed in phase `C Compiler'. (Exit code: 1)
```

--- a/abcBridge.cabal
+++ b/abcBridge.cabal
@@ -77,18 +77,6 @@ library
     -- -U__BLOCKS__ to work around https://github.com/haskell/c2hs/issues/29
     -- has_attribute and has_features undefined for similar reasons...
     CC-options: -U__BLOCKS__ -U__has_attribute -U__has_feature
-
-    -- Also, because abc lib has a bogus stdint.h in the
-    -- include/sat/glucose/ directory that will trip up cpp2hs if it
-    -- encounters it, inclusion of that file needs to be avoided.
-    -- This does not appear to be an issue for Linux or Windows, but
-    -- the stdint.h file is included by stdlib.h on Darwin... but only
-    -- if __DARWIN_C_LEVEL >= __DARWIN_C_FULL (see <cdefs.h>).  The
-    -- __DARWIN_C_FULL defined level is 900000L indicating full Darwin
-    -- support levels, but abc library should only need ANSI C,
-    -- achieved by defining _ANSI_SOURCE which causes __DARWIN_C_LEVEL
-    -- to be defined at 010000L and thus avoid the stdint.h include.
-    CC-options: -D_ANSI_SOURCE
   }
 
   if os(windows) {

